Dockerfile.develop

```diff
@@ -0,0 +1,45 @@
+FROM node:10-alpine
+
+# Support custom branches of the react-sdk and js-sdk. This also helps us build
+# images of riot-web develop.
+ARG USE_CUSTOM_SDKS=false
+ARG REACT_SDK_REPO="https://github.com/matrix-org/matrix-react-sdk.git"
+ARG REACT_SDK_BRANCH="master"
+ARG JS_SDK_REPO="https://github.com/matrix-org/matrix-js-sdk.git"
+ARG JS_SDK_BRANCH="master"
+
+# CRLF -> LF conversion
+RUN apk add --no-cache git dos2unix
+
+WORKDIR /src
+
+# Install deps-only into separate Docker layer and run Riot when the container
+# starts, mount source as a volume and keep dev environment consistent
+COPY scripts /src/scripts
+RUN dos2unix /src/scripts/docker-link-repos.sh \
+    && sh /src/scripts/docker-link-repos.sh
+
+# Cache package.json
+COPY package.json yarn.lock /src/
+
+# Remove 'prepare' script because there is NO config in yarn and npm for
```

Comment:
meh - this is fine tbh. It shouldn't cause problems, it's just a waste of cycles. 

tbh I'm more inclined to leave it there than rip it out for future maintenance things.

Reply:
I've tried removing it again today and it throws this error:

```bash
Step 12/15 : RUN yarn --network-timeout=100000 install
 ---> Running in a026b8e776e8
yarn install v1.17.3
[1/4] Resolving packages...
[2/4] Fetching packages...
warning Pattern ["gemini-scrollbar@matrix-org/gemini-scrollbar#b302279"] is trying to unpack in the same destination "/usr/local/share/.cache/yarn/v4/npm-gemini-scrollbar-1.4.3/node_modules/gemini-scrollbar" as pattern ["gemini-scrollbar@github:matrix-org/gemini-scrollbar#b302279"]. This could result in non-deterministic behavior, skipping.
info fsevents@1.2.9: The platform "linux" is incompatible with this module.
info "fsevents@1.2.9" is an optional dependency and failed compatibility check. Excluding it from installation.
[3/4] Linking dependencies...
warning "matrix-react-sdk > emojibase-data@4.0.1" has unmet peer dependency "emojibase@*".
warning "matrix-react-sdk > slate@0.41.3" has unmet peer dependency "immutable@>=3.8.1".
warning "matrix-react-sdk > slate-html-serializer@0.6.32" has unmet peer dependency "immutable@>=3.8.1".
warning "matrix-react-sdk > slate-md-serializer@3.1.0" has unmet peer dependency "immutable@>=3.0.0".
warning "matrix-react-sdk > slate-react@0.18.11" has unmet peer dependency "immutable@>=3.8.1".
warning "matrix-react-sdk > diff-dom > rollup-plugin-terser@4.0.4" has unmet peer dependency "rollup@>=0.66.0 <2".
warning "matrix-react-sdk > slate-react > react-immutable-proptypes@2.1.0" has unmet peer dependency "immutable@>=3.6.2".
warning "matrix-react-sdk > slate-react > slate-plain-serializer@0.6.39" has unmet peer dependency "immutable@>=3.8.1".
warning "matrix-react-sdk > slate-react > slate-prop-types@0.4.67" has unmet peer dependency "immutable@>=3.8.1".
warning " > electron-builder-squirrel-windows@21.2.0" has unmet peer dependency "app-builder-lib@~21.2.0".
[4/4] Building fresh packages...
$ yarn clean && yarn build:compile
yarn run v1.17.3
$ rimraf lib webapp electron_app/dist
Done in 0.24s.
yarn run v1.17.3
$ yarn reskindex && babel --source-maps -d lib src
$ reskindex -h src/header
src doesn't exist
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
The command '/bin/sh -c yarn --network-timeout=100000 install' returned a non-zero code: 2
```

Reply:
Eh, this is embarrassing... for some reason it never occurred to me it's a file ^_^" Fixed!